src/rules/function-linear-gradient-no-nonstandard-direction/index.js

```diff
@@ -41,7 +45,8 @@ export default function (actual) {
         // cf. https://drafts.csswg.org/css-images-3/#linear-gradient-syntax
         if (!/left|right|top|bottom/.test(firstArg)) { return }
 
-        if (!isStandardDirection(firstArg)) {
+        const withToPrefix = !/^-webkit-|-moz-|-o-/.test(decl.value)
```

Comment:
@stylelint/core Would we want to use `postcss`'s `vendor.prefix()` here for the sake of consistency with other rules?